src/unix/notbsd/linux/other/b32/x86.rs

```diff
@@ -18,3 +18,53 @@ pub const SO_SNDTIMEO: ::c_int = 21;
 
 pub const FIOCLEX: ::c_ulong = 0x5451;
 pub const FIONBIO: ::c_ulong = 0x5421;
+
+type greg_t = ::c_int;
+
+const NREG : usize = 19;
+
+type gregset_t = [greg_t; NREG];
+
+s! {
```

Comment:
Can you include these definitions and such in the same style as the rest of the file? For example typdefs are at the top, then the structs, then constants, etc.